pkg/metricsstore/metricsstore.go

```diff
@@ -49,11 +49,11 @@ type MetricsStore struct {
 	/*
 	 * Certificate metrics
 	 */
-	// CertXdsIssuedCount is the metric counter for the number of xds certificates issued
-	CertXdsIssuedCount prometheus.Counter
+	// CertIssuedCount is the metric counter for the number of certificates issued
+	CertIssuedCount prometheus.Counter
 
-	// CertXdsIssuedCounter the histogram to track the time to issue xds certificates
-	CertXdsIssuedTime *prometheus.HistogramVec
+	// CertXdsIssuedCounter the histogram to track the time to issue a certificates
+	CertIssuedTime *prometheus.HistogramVec
```

Comment:
These metrics only pertain to XDS certs, why are they being removed from the variable names?

Reply:
sorry, forgot to mention the change.
The current implementation actually only tracks the injector ones, the SDS part requires fixing (#2606). 
Since injector and osm can use the same metric, i just renamed it to something bit more generic (updated the doc too). For metric purposes, injector and xds cert creations will still be separate by labeling and origin, even when sharing the same metric.